tidy3d/log.py

```diff
@@ -330,15 +330,15 @@ def set_logging_file(
     if "file" in log.handlers:
         try:
             log.handlers["file"].file.close()
-        except:  # pylint: disable=bare-except
+        except:  # noqa: E722
             log.warning("Log file could not be closed")
         finally:
             del log.handlers["file"]
 
     try:
         # pylint: disable=consider-using-with
         file = open(fname, filemode)
-    except:  # pylint: disable=bare-except
+    except:  # noqa: E722
```

Comment:
so pylint disables still work? but there's also another way? how does it work exactly?

Reply:
Another option is to just add ruff in addition to pylint and then talk about pydantic v2 later.. ruff might be nice to have and we could try it out a bit before transitioning completely?

Reply:
What is the advantage of leaving pylint but *also* using ruff?

Reply:
* ruff catches different stuff, seemingly a wider range of issues.
* could test it out before phasing out pylint.
but yea if having two tools just complicates our lives I dont know